src/store/middleware/view.tsx

```diff
@@ -23,4 +23,16 @@ export const passViewMiddleware = (stateAPI: MiddlewareAPI<Dispatch, Application
         });
       });
   }
+
+  if (action.type === Action.EnableHotkeyNotifications) {
+    if (typeof window !== undefined) {
+      window.localStorage.setItem("hotkeyNotificationsEnabled", JSON.stringify(true));
```

Comment:
all locale storage keys have `"scrumlr/"` as prefix (except `theme`), maybe you should add it for consistency